vxyowsup/whatsapp.py

```diff
@@ -154,8 +154,8 @@ def onMessage(self, messageProtocolEntity):
             messageProtocolEntity.getParticipant())
         self.toLower(receipt)
 
-        print "You have received a message, and thusly sent a receipt"
-        print "You are now sending a reply"
+        log.debug('You have received a message, and thusly sent a receipt')
+        # log.debug("You are now sending a reply")
         # self.send_to_human('iiii', from_address + '@s.whatsapp.net', "this transport's gone rogue")
```

Comment:
Ahh, I think I was misunderstanding where this code would be. Yeah this sounds good. I'll just put it into this PR, I think we're still under 8 hours :smile: